test/e2e/setup/setup.go

```diff
@@ -126,37 +108,41 @@ func GeneratePasswordForUser(mdbu mdbv1.MongoDBUser, ctx *f.Context, namespace s
 		SetField(passwordKey, password).
 		Build()
 
-	return password, f.Global.Client.Create(context.TODO(), &passwordSecret, &f.CleanupOptions{TestContext: ctx})
+	return password, e2eutil.TestClient.Create(context.TODO(), &passwordSecret, &e2eutil.CleanupOptions{TestContext: ctx})
 }
 
-func deployOperator(c f.FrameworkClient) error {
+func deployOperator(c client.Client) error {
 	testConfig := loadTestConfigFromEnv()
 
-	if err := buildKubernetesResourceFromYamlFile(c, path.Join(deployDir, "role.yaml"), &rbacv1.Role{}, withNamespace(testConfig.namespace)); err != nil {
+	e2eutil.OperatorNamespace = testConfig.namespace
+	fmt.Printf("Setting operator namespace to %s\n", e2eutil.OperatorNamespace)
+	if testConfig.clusterWide {
+		e2eutil.WatchNamespace = "*"
```

Comment:
[q] do we need an `e2eutil.WatchNamespace` package level field? It looks like it could be replaced with a function level scoped variable (I only see it being used in `deployOperator`)

Reply:
Done!